src/post-message.ts

```diff
@@ -292,6 +297,12 @@ const isDisabledFromIframe = () => {
     return !window?.frameElement?.hasAttribute('data-enable-webview-bridge');
 };
 
+let log: undefined | ((...args: Array<any>) => void) = undefined;
+
+export const setLogger = (logger: typeof log): void => {
+    log = logger;
+};
```

Comment:
we can use this to set a logger function like `console.debug` or `logBreadcrumb` to log the bridge traffic